examples/apollo/package.json

```diff
@@ -8,7 +8,12 @@
     "start": "cosmos"
   },
   "dependencies": {
-    "react-apollo": "^1.4.15",
+    "apollo-cache-inmemory": "^1.1.4",
```

Comment:
Does the consuming repo need this dependency?

Reply:
Surely not! Thanks! 🎩